### PR TITLE
Dl3 adding card_network def

### PR DESCRIPTION
**Pull request by dallaslones**

### CODE REVIEW

File: lib/cashbox/model/payment_method.rb [hunk 1]
```diff
@@ -28,5 +28,13 @@ def direct_debit?
     def last_digits
       self[type.underscore].last_digits
     end
+
+    def card_network
+      return "not a card" unless type == "CreditCard"
```
Comment (maulin) [inline lib/cashbox/model/payment_method.rb]: should throw an error for this

File: lib/cashbox/model/payment_method.rb [hunk 2]
```diff
@@ -28,5 +28,13 @@ def direct_debit?
...
+      bin_number = credit_card.bin
+      return "Visa" if bin_number.start_with?("3")
+      return "Master Card" if bin_number.start_with?("2", "5")
+      "unknown network"
```
Comment (maulin) [inline lib/cashbox/model/payment_method.rb]: can return `nil` instead of "unknown card"

File: lib/cashbox/model/payment_method.rb [hunk 3]
```diff
@@ -28,5 +28,13 @@ def direct_debit?
     def last_digits
       self[type.underscore].last_digits
     end
+
+    def card_network
+      return nil unless type == "CreditCard"
+      bin_number = credit_card.bin
+      return "Visa" if bin_number.start_with?("3")
+      return "Master Card" if bin_number.start_with?("2", "5")
+      nil
```
Comment (maulin) [inline lib/cashbox/model/payment_method.rb]: ruby methods return nil by default. The first line can be `return unless type == "CreditCard`

Comment (julamula) [reply lib/cashbox/model/payment_method.rb]: 😆 @maulin aren't you the one that told him to return nil?

**Pull request closed without merging**